src/stack.py

```diff
@@ -0,0 +1,28 @@
+"""This is the module to push and pop off a stack."""
+from linked_list import LinkedList
+
+
+class Stack(object):
+    """Define stack class."""
+
+    def __init__(self, iterable=None):
+        """.Initiate the Stack as a instance of LinkedList."""
+        self._container = LinkedList(iterable)
+
+    def pop(self):
+        """Remove the top element from stack and returns it."""
+        self._container.pop()
+
+    def push(self, val):
+        """Add a value to the top of the stack."""
+        self._container.push(val)
+        return self._container
+
+    def _size(self):
```

Comment:
If this is a hidden method, what's the thing we should use to actually get this value out on a regular basis? Maybe include a `__len__` method?